lib/recurly.js

```diff
@@ -229,6 +260,12 @@ export class Recurly extends Emitter {
       this.emit('configured');
       this.report('configured');
     }
+
+    // Since this readyState indicates synchronous preparation, we may now
+    // emit ready
+    if (this.readyState === 3) {
+      this.emit('ready');
+    }
```

Comment:
This new action is to become the default result when using `Elements`. Eventually, we'll be able to do away with readyState tracking altogether.